src/PIL/Jpeg2KImagePlugin.py

```diff
@@ -192,7 +192,7 @@ def _open(self):
                 length = -1
 
         self.tile = [('jpeg2k', (0, 0) + self.size, 0,
-                      (self.codec, self.reduce, self.layers, fd, length, self.fp))]
+                      (self.codec, self.reduce, self.layers, fd, length))]
```

Comment:
Looking back through the history, I know that this was done in the rework of adding pulls_fp to remove threading from Jpeg2k, but what I don't see is if this was just an initial cut that got replaced with a different version and didn't get updated. I'd say that it's likely ok if the Jpeg2k tests are all passing, as it's clearly not undoing the whole PR and if something like that was really necessary, things would fall over everywhere.